### DISCUSSION TITLE
Error when accessing session api

### DISCUSSION THREAD

**Discussion by bakerhaider** [swellstores/community#444] [Help] [answered]
`const loggedUser = async () => {
        const res = await swell.session.get()
        console.log(res);
      }`
I get the following VSCode error in the .session property: Property 'session' does not exist on type 'typeof import("c:/Users/Sigmar/Desktop/Projectz/playgroud/swll/node_modules/swell-js/types/index")'.ts(2339)

**Comment by nathanlevitt** [comment]
@logeshswell are session endpoints down? Querying `/:sessions` gives empty response. Also, getting `null` response for any `/account` endpoint with valid `X-Session` header.

**Reply by bakerhaider** [reply]
Hi can you help with another issue? thanks

**Reply by ericingram** [reply, marked as answer]
As a followup, this was corrected last week.

**Reply by bakerhaider** [reply]
Hello, are you able to help me with another issue i have?

**Reply by logeshswell** [reply]
Please share the issue with details and folks around you will help you as soon as they notice.

**Reply by vettloffah** [reply]
Hello I'm sending a GET request on `/:sessions` and getting empty results array:
```js
{ count: 0, page_count: 1, page: 1, results: [] }
```
I'm sure there are active sessions of logged in users.
It's also returning this from console in the swell dashboard. Is this broken or is there another way to get active session data?

**Reply by logeshswell** [reply]
There's no way to fetch a list of all active logged-in sessions from the frontend/backend APIs.
But, it is possible to fetch the details of the current session using frontend API as follows:
https://developers.swell.is/frontend-api/session-management#get-session-values

**Comment by vettloffah** [comment]
I have a request coming from front end to a backend API route and I need
to verify that the request is coming from a specific set of user accounts.

I can send the session cookie to backend, but then I can’t seem to be able
to use the API to verify which account that session belongs to.

**Reply by logeshswell** [reply]
Can you the following docs:
https://developers.swell.is/frontend-api/session-management#get-logged-in-account
https://developers.swell.is/frontend-api/session-management#get-session-values

It returns the account_id and account information of the session cookie that you sent to backend.
